GSG-G7/mental-snapp
# Journal Card

issue: #73 **Journal Card**

## What did I do?
- I made the layout for the journal card
- I made it ready for us to use props with it(add data to it)
- I made it responsive
- I downloaded the **Moment** module to help us format the time
- I wrote the shape of the object which comes or should come :smile: from the DB
- I wrote a storybook for the component so you can see it 

## How to test?
Just run `npm run storybook` and it should be with the components

**Note**: You can edit the object and work with moment to understand it
